clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -239,6 +239,9 @@ const PrivacyRequestForm: React.FC<PrivacyRequestFormProps> = ({
                   onChange={handleChange}
                   onBlur={handleBlur}
                   value={values.email}
+                  disabled={Boolean(
```

Comment:
It looks like this should use the `isDisabled` prop instead: https://chakra-ui.com/docs/components/input/props

I only noticed this because of the type change in the `PhoneInput` component, by the way - very subtle but I assume that Chakra has good reasons for that pattern

Reply:
addressed